openssl-dynamic/src/main/c/error.c

```diff
@@ -49,6 +49,19 @@ void tcn_ThrowException(JNIEnv *env, const char *msg)
 
 }
 
+void tcn_ThrowIllegalArgumentException(JNIEnv *env, const char *msg)
+{
+    jclass javaExceptionClass;
+
+    javaExceptionClass = (*env)->FindClass(env, "java/lang/IllegalArgumentException");
+    if (javaExceptionClass == NULL) {
+        fprintf(stderr, "Cannot find java/lang/IllegalArgumentException class\n");
```

Comment:
@trustin yes! Let me actual cache the class during init and if we can not find it fail the init :)